kevm-pyk/src/kevm_pyk/utils.py

```diff
@@ -82,6 +82,7 @@ def kevm_prove(
     break_on_calls: bool = True,
     extract_branches: Callable[[CTerm], Iterable[KInner]] | None = None,
     abstract_node: Callable[[CTerm], CTerm] | None = None,
+    max_branches: int | None = 1,
```

Comment:
To preserve backward compatibility for now:

```suggestion
    max_branches: int | None = None,
```

Reply:
It's now an option available for the `foundry-prove` command, and is None by default.